shared-ui/components/primary-button/PrimaryButtonWide.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { StyledPrimaryButtonWide } from './PrimaryButton.styles';
+import { ButtonProps } from '../../lib/types';
+import { buttonAnimations } from './PrimaryButton.animations';
+
+const PrimaryButton: React.FC<ButtonProps> = ({
```

Comment:
This PrimaryButton seems pretty identical to the original one so instead of creating this button, could we pass in a prop to the original button that's like "isSmall" or something and then add the extra padding based on that? 

Like this where we conditionally set the design based on the prop - this would get rid of a lot of code I think!
<img width="581" alt="Screenshot 2023-01-05 at 6 22 30 PM" src="https://user-images.githubusercontent.com/59738880/210898882-2955f978-f632-470c-983f-935fb6e8b09c.png">